src/third-party/PhishingController.ts

```diff
@@ -23,6 +23,43 @@ export interface EthPhishingResponse {
   whitelist: string[];
 }
 
+/**
+ * @type EthPhishingDetectConfig
+ *
+ * Interface defining expected input to PhishingDetector.
+ * @property allowlist - List of approved origins (legacy naming "whitelist")
+ * @property blocklist - List of unapproved origins (legacy naming "blacklist")
+ * @property fuzzylist - List of fuzzy-matched unapproved origins
+ * @property tolerance - Fuzzy match tolerance level
+ */
+export interface EthPhishingDetectConfig {
+  allowlist: string[];
+  blocklist: string[];
+  fuzzylist: string[];
+  tolerance: number;
+  name: string;
+  version: number;
+}
+
+/**
+ * @type EthPhishingDetectResult
+ *
+ * Interface that describes the result of .
+ * @property name - Name of the config on which a match was found. Not returned when using legacy config.
+ * @property version - Version of the config on which a match was found. Not returned when using legacy config.
+ * @property result - Whether a domain was detected as a phishing domain. True means an unsafe domain.
+ * @property match - The matching fuzzylist origin when a fuzzylist match is found. Returned as undefined for non-fuzzy true results.
+ * @property type - The field of the config on which a match was found. 'whitelist' and 'blacklist' can be returned when using legacy config.
+ */
+export interface EthPhishingDetectResult {
+  name?: string; // Not returned for legacy config.
+  version?: string; // Not returned for legacy config.
+  result: boolean;
+  match?: string;  // Returned as undefined for non-fuzzy true results.
+  type: 'all' | 'fuzzy' | 'blocklist' | 'allowlist' | 'whitelist' | 'blacklist';
```

Comment:
This can be updated now that the legacy config is no longer used. Also the description seemed unfinished so I completed the sentence.

```suggestion
 * Interface that describes the result of the `test` method.
 * @property name - Name of the config on which a match was found.
 * @property version - Version of the config on which a match was found.
 * @property result - Whether a domain was detected as a phishing domain. True means an unsafe domain.
 * @property match - The matching fuzzylist origin when a fuzzylist match is found. Returned as undefined for non-fuzzy true results.
 * @property type - The field of the config on which a match was found.
 */
export interface EthPhishingDetectResult {
  name: string;
  version: string;
  result: boolean;
  match?: string;  // Returned as undefined for non-fuzzy true results.
  type: 'all' | 'fuzzy' | 'blocklist' | 'allowlist';
```

Reply:
Sorry about the incomplete sentences, there was also a previous mistake of mine that you caught.